app/js/actions/patient.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import { SET_PATIENT, SET_NOTE } from './actionTypes';
+
+const applicationDistribution = window.location.href.split('/')[3];
+const apiBaseUrl = `/${applicationDistribution}/ws/rest/v1`;
+
+/**
+ * @description - fetch patient information
+ *
+ * @export
+ *
+ * @param {string} uuid - User unique ID
+ *
+ * @returns {object} dispatch - Dispatched action
+ */
+export function fetchPatientRecord(uuid) {
+  return dispatch => axios.get(`${apiBaseUrl}/patient/${uuid}?v=custom:(patientId,uuid,patientIdentifier:(uuid,identifier),person:(gender,age,birthdate,birthdateEstimated,personName,preferredAddress),attributes:(value,attributeType:(name)))`)
+    .then((response) => {
+      dispatch({
+        type: SET_PATIENT,
+        patient: response.data,
+      });
+    })
+    .catch((error) => {
+      if (error.response) {
+        // TODO uncomment this to redirect user to homepage if uuid not present
+        // location.href = `/${applicationDistribution}`;
```

Comment:
When do you plan to address the above?

Reply:
This will be addressed when OEUI-40 is done, thanks